src/main/java/com/example/java/test/junior/developer/keycloak/KeycloakAdminClient.java

```diff
@@ -16,7 +19,17 @@ public class KeycloakAdminClient {
 
   public void createUser(UserRepresentation user) {
     log.info("Creating user with username: {}", user.getUsername());
-    keycloak.realm(keycloakProperties.getRealm()).users().create(user);
-    log.info("User created successfully");
+    var response = keycloak.realm(keycloakProperties.getRealm()).users().create(user);
+    int statusCode = response.getStatus();
+    if (statusCode == Response.Status.CREATED.getStatusCode()) {
+      log.info("User created successfully with status code: {}", statusCode);
+    } else {
+      log.error("User creation failed with status code: {}", statusCode);
+      try {
+        throw new RuntimeException("User creation failed");
```

Comment:
What is the purpose of this line? 
You throw exception then immediately clutch it on the next line. It doesn't make any sense...

Reply:
Yes, it doesn't make sense now. Just a remnant of previously written code. I don't know why I didn't delete it.